projects/aca-shared/rules/src/navigation.rules.spec.ts

```diff
@@ -225,6 +225,38 @@ describe('navigation.evaluators', () => {
     });
   });
 
+  describe('isDetails', () => {
+    it('should return true if url ends with `/details`', () => {
+      const context: any = {
+        navigation: {
+          url: '/path/details'
+        }
+      };
+
+      expect(app.isDetails(context)).toBe(true);
+    });
+
+    it('should return true if url starts with `/details`', () => {
+      const context: any = {
+        navigation: {
+          url: '/details/path'
+        }
+      };
+
+      expect(app.isDetails(context)).toBe(true);
+    });
+
+    it('should return true if url includes with `/details`', () => {
```

Comment:
logic of that test is the same like logic of above test

Reply:
I have updated the test cases